src/Login.css

```diff
@@ -0,0 +1,63 @@
+.app {
+    font-family: sans-serif;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    gap: 20px;
+    height: 100vh;
+    font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+    background-color: #f8f9fd;
```

Comment:
We also need to confirm these, since wireframes and Figma haven't been confirmed yet -- maybe just add a TODO comment:
//TODO: confirm once mockups provided